core/common/src/main/java/alluxio/wire/InconsistentProperty.java

```diff
@@ -104,10 +105,25 @@ public int hashCode() {
   public String toString() {
     return Objects.toStringHelper(this)
         .add("key", mName)
-        .add("values", mValues)
+        .add("values", getValuesWithoutOptionalKeyword())
         .toString();
   }
 
+  /**
+   * @return the string of mValues without Optional keyword
+   */
+  private String getValuesWithoutOptionalKeyword() {
+    String readableNullValue = "no value set";
+    String valueFormat = "%s (%s)";
```

Comment:
can we inline these two string constants? I don't think the names are important enough to be worth the extra space and distance from usage.